test/Project.toml

```diff
@@ -0,0 +1,13 @@
+[deps]
```

Comment:
We currently define the test dependencies in the main Project.toml. I'm not sure why this needs to be added.

Reply:
I added it because I find it is convenient for testing individual test files locally, but I can remove it (also I thought that was the modern way of specifying test dependencies anyway).